src/remote/remote.cpp

```diff
@@ -855,6 +856,87 @@ ISC_STATUS* Rdb::get_status_vector() throw()
 }
 */
 
+
+bool RBlobInfo::getLocalInfo(unsigned int itemsLength, const unsigned char* items,
+	unsigned int bufferLength, unsigned char* buffer)
+{
+	if (!valid)
+		return false;
+
+	unsigned char* p = buffer;
+	const unsigned char* const end = buffer + bufferLength;
+
+	for (auto item = items; p && (item < items + itemsLength); item++)
+	{
+		switch (*item)
+		{
+		case isc_info_blob_num_segments:
+			p = fb_utils::putInfoItemInt(*item, num_segments, p, end);
+			break;
+
+		case isc_info_blob_max_segment:
+			p = fb_utils::putInfoItemInt(*item, max_segment, p, end);
+			break;
+
+		case isc_info_blob_total_length:
+			p = fb_utils::putInfoItemInt(*item, total_length, p, end);
+			break;
+
+		case isc_info_blob_type:
+			p = fb_utils::putInfoItemInt(*item, blob_type, p, end);
+			break;
+
+		case isc_info_end:
+			if (p < end)
+				*p++ = isc_info_end;
+			break;
+
+		default:
+			// unknown info item, let remote server handle it
+			return false;
+		}
+	}
+
+	return true;
+}
+
+
+void RBlobInfo::parseInfo(unsigned int bufferLength, unsigned char* buffer)
```

Comment:
May be better
const unsigned char* buffer
?

Reply:
Sure, thanks